scripts/upgrade_v01.7.py

```diff
@@ -0,0 +1,22 @@
+import os
+import pymongo
+
```

Comment:
I would like some code commenting here. Why are we doing this? (I know why of course, but will future-me remember?